sdk/stats.js

```diff
@@ -0,0 +1,29 @@
+const urljoin = require('urljoin');
+const logger = require('../lib/logger');
+const { loggedRequest } = require('./handleError');
+
+const { getDeployEndpoint } = require('./config');
+
+/**
+ * Retrieves account usage statistics
+ *
+ * @param {string} apiKey - Binaris API key
+ * @param {Date} since - All stats since the date-time
+ * @param {Date} until - All stats until the date-time
+ */
+const stats = async function stats(apiKey, since, until) { // eslint-disable-line consistent-return,max-len
```

Comment:
Micro-style comment: the `// eslint-disable-line` seems obsolete.

Also could be written as `async function stats` without `const stats =` to avoid `Foo foo` repetition.